Screens/History.js

```diff
@@ -53,15 +44,15 @@ class HistoryItems extends React.Component {
   //retrieving everything from the database, then putting them into an array and storing it in the state
   update() {
     hdb.transaction((tx) => {
-      tx.executeSql(
-        `select * from historyitems;`,
-        [],
-        (_, { rows: { _array } }) => this.setState({ historyItems: _array })
-      );
+      tx.executeSql(`select * from historyitems;`, [], (_, { rows: { _array } }) => this.setState({ historyItems: _array }));
     });
   }
 }
-
+const handleDeleteBtn = () => {

```

Comment:
If I'm understanding this correctly, here once the handleDeleteBtn is mapped to a button, it will go to the history database and execute the command on line 53 which will delete everything stored

Reply:
That's right! This function is called when the delete button is pressed.